h/templates/accounts/developer.html.jinja2

```diff
@@ -5,25 +5,55 @@
 
 {% block page_content %}
   <div class="form-vertical">
+    {% if not feature('activity_pages') -%}
     <h2 class="form-heading">
       <span>{% trans %}Generate your API token{% endtrans %}</span>
     </h2>
+    {% endif %}
 
     {% if token %}
+      {% if feature('activity_pages') %}
+        <p class="form-help-text">
+        <label for="token">{% trans %}Your API token is:{% endtrans %}</label>
+        </p>
+        <input id="token"
+               type="text"
+               value="{{ token }}"
+               readonly="readonly"
+               class="api-token">
+        <p class="help-block form-help-text">
+          Please keep your API token safe as it can be used to access your account.
+        </p>
+      {% else %}
       <label for="token">{% trans %}Your API token is:{% endtrans %}</label>
       <input id="token" type="text" value="{{ token }}" readonly="readonly"
              class="api-token-input">
       <span class="help-block">
         Please keep your API token safe as it can be used to access your
         account.
       </span>
+      {% endif %}
     {% else %}
+      {% if not feature('activity_pages') -%}
       <legend>
         {% trans %}You haven't generated an API token yet.{% endtrans %}
       </legend>
+      {%- endif %}
     {% endif %}
 
       <form method="POST">
+        {% if feature('activity_pages') %}
+          {% if token %}
+            <button type="submit" class="btn btn-danger"
```

Comment:
`btn-danger` doesn't exist in the new styles (it would be `btn--danger` as a variant of btn). Since the styling isn't any different for the "Regenerate your token" button we can just remove this class.